activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-liquibase/src/main/resources/config/modeling/liquibase/master.xml

```diff
@@ -8,46 +8,46 @@
   <changeSet author="aae-modeling"
              id="h2-schema-creation" dbms="h2">
     <sqlFile dbms="h2"
-             encoding="utf8"
```

Comment:
Could you please revert the formatting changes to the file? The git history and the PR would be cleaner.

Reply:
Done.